docs/celsius-converter-exception/index.md

```diff
@@ -38,19 +19,13 @@ But beware, because the compiler knows nothing about the types of those fields.
 For example, we could write `ReactEvent.Form.target(evt)##value + 1`, treating
 it as if it were an integer, and the compiler wouldn't complain.
 
+## Wrap functions that return Js.t
```

Comment:
```suggestion
## Wrap functions that return `Js.t`
```
or maybe "a `Js.t` value"?

Reply:
When possible, I try to keep the subsection headers short so that they aren't truncated in the sidebar